egs/libri_trans/asr1/conf/tuning/train_pytorch_transformer_bpe_long.yaml

```diff
@@ -0,0 +1,44 @@
+# network architecture
+# encoder related
+elayers: 12
+eunits: 2048
+# decoder related
+dlayers: 6
+dunits: 2048
+# attention related
+adim: 256
+aheads: 4
+
+# hybrid CTC/attention
+mtlalpha: 0.3
+
+# label smoothing
+lsm-weight: 0.1
+
+# minibatch related
+batch-size: 64
+maxlen-in: 512  # if input length  > maxlen-in, batchsize is automatically reduced
+maxlen-out: 150 # if output length > maxlen-out, batchsize is automatically reduced
+
+# optimization related
+sortagrad: 0 # Feed samples from shortest to longest ; -1: enabled for all epochs, 0: disabled, other: enabled for 'other' epochs
+opt: noam
+accum-grad: 2
+grad-clip: 5
+patience: 0
+epochs: 120
```

Comment:
The only difference is the number of epochs? Then, we could just rename `train_pytorch_transformer_bpe_long.yaml ` as `train_pytorch_transformer_bpe.yaml` and replace it.

Reply:
I'd like to keep as they are in order to keep the training time for the baseline without SpacAugment.

Reply:
Oh, now I got it. This longer version is for the specaug, right? Then, maybe better to add some comments in `run.sh`? Another suggestion is to use specaug as default in `run.sh`.

Reply:
I don't wanna use SpecAugment as the default setting. I added a comment in `run.sh`. Also, readers can easily follow it by looking at `RESULTS.md`.

Reply:
ok. thanks!